include/mc_tasks/CoPTask.h

```diff
@@ -97,6 +97,13 @@ struct MC_TASKS_DLLAPI CoPTask : DampingTask
       double dt,
       const mc_rtc::Configuration & config) const override;
 
+  /**
+   * @brief Wether to compute the desired torque with the target pressure (true) of the measured one (false)
+   *
+   * @param s
+   */
+  void setUsedPressure(bool s) { useTargetPressure = s; }
```

Comment:
I suggest to rename this:

```suggestion
  void useTargetPressure(bool s) { useTargetPressure_ = s; }
```

I think it should also:
- have an accessor
- be settable via the YAML configuration